install.sh

```diff
@@ -24,13 +21,9 @@ echo 'IPAPPEND=3' >> /etc/ltsp/update-kernels.conf
 /usr/share/ltsp/update-kernels
 
 # Configure dnsmasq
+echo "port=5353" >> /etc/dnsmasq.conf
```

Comment:
Why is this needed?

Reply:
See the issue is in `tara` dnsmasq is conflicting with systemd-resolved. It gives this error
```
dnsmasq[4285]: failed to create listening socket for port 53: address already in use
```
As a result dnsmasq service is unable to start. So a good option was to switch it to another port.

Reply:
Yes. I saw that one as well. At some point it involves turning on and off the wired connection with `nmcli`. Doing that just freezes vagrant box. 
It might work for actual  or virtual machine. But not for vagrant box